includes/wikia/services/AvatarService.class.php

```diff
@@ -269,7 +269,7 @@ public static function getVignetteUrl( Masthead $masthead, $width, $timestamp )
 				$url = self::buildVignetteUrl( $width, $bucket, $relativePath, $timestamp, false );
 			}
 		} else { // default avatar
-			$legacyDefaultUrl = array_shift( $masthead->getDefaultAvatars( 'thumb/' ) );
+			$legacyDefaultUrl = @array_shift( $masthead->getDefaultAvatars( 'thumb/' ) );
```

Comment:
Using @ is deemed to be a bad practice. Can we handle this in some other way or at least add a comment saying explicitly what is the case we want to hide warnings for?